docs/challenges/reattempt-challenge.mdx

```diff
@@ -1,10 +1,12 @@
 ---
 title: "Can I re-attempt a challenge in a different language?"
-description: "Yes, you can! Use the dropdown on the top-right to do this:"
+description: ""
 ---
 
+Yes! From the dropdown menu in the sidebar, select **Try a different language**.
+
+You can switch between languages whenever you want.
+
 <Frame>
   <img className="w-full" src="/img/reattempt.png" />
 </Frame>
-
-You can switch between languages whenever you want.
```

Comment:
@glokta1 this'd be a good place to say something about re-attempting using the same language? Something like: 

> You can also re-attempt using the same language if you'd like. Your old repository will remain intact, and you can switch back to it at any time.

Reply:
(Ah, just saw your PR description - yep let's somehow incorporate the "re-attempt with same language" information)